src/tools/compiletest/src/runtest.rs

```diff
@@ -3399,7 +3399,9 @@ impl<'test> TestCx<'test> {
 
         for output_file in &files {
             if actual.is_empty() {
-                self.delete_file(output_file);
+                if output_file.exists() {
+                    self.delete_file(output_file);
+                }
```

Comment:
This is an unrelated change, right? Seems reasonable though. (Still technically a race condition though, would be safer to just "delete and ignore errors", but whatever.)

Reply:
Yea, I didn't want to touch the `delete_file` function, but I'll do that now